src/Facility.LanguageServer/FsdDefinitionUtility.cs

```diff
@@ -22,9 +22,9 @@ private static string GetMemberTypeName(this ServiceTypeInfo type)
 			switch (type.Kind)
 			{
 				case ServiceTypeKind.Dto:
-					return type.Dto.Name;
+					return type.Dto?.Name;
 				case ServiceTypeKind.Enum:
-					return type.Enum.Name;
+					return type.Enum?.Name;
```

Comment:
Is there a specific literal we would want to return in the cases that these are null?

Reply:
It would be an error for the values to be null in this case, so `!` would be appropriate.